ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -97,7 +103,8 @@ def _llmobs_set_tags(
         elif operation == "tool":
             self._llmobs_set_meta_tags_from_tool(span, tool_inputs=kwargs, tool_output=response)
 
-        span.set_tag_str(METRICS, safe_json({}))
+        if span.get_tag(METRICS) is None:
+            span.set_tag_str(METRICS, safe_json({}))
```

Comment:
Do we need to set metrics if empty? If not, I think we can just get rid of this.

Reply:
good question, probably not? I had just adjusted the logic from what we had before. I'll remove and see if it breaks tests

Reply:
looks like it's fine to remove